src/i18n/index.ts

```diff
@@ -1 +1,21 @@
-export * from './fr'
+import { useSelector } from 'react-redux'
+import { selectLocale } from '../core/useCases/locale'
+import { Locale } from '../core/useCases/locale/locale.reducer'
+import { Texts, texts as frTexts } from './fr'
+
+const langs: Record<Locale, Texts> = {
+  fr: frTexts,
+  en: frTexts,
+}
```

Comment:
Du coup même pas besoin de `en` tant que les textes n'existent pas

```suggestion
const langs: Record<Locale, Texts> = {
  fr: frTexts,
}
```